src/components/login/LoginForm.tsx

```diff
@@ -1,30 +1,36 @@
+import { useForm, SubmitHandler } from "react-hook-form";
+import InputFieldLogin from "./InputFieldLogin";
+import { loginUser } from "../../services/userService";
```

Comment:
You can investigate about relative & absolute imports to avoid the `"../../"`
https://medium.com/geekculture/making-life-easier-with-absolute-imports-react-in-javascript-and-typescript-bbdab8a8a3a1